### ISSUE TITLE
Changed semantic of `Offset` variable

### ISSUE THREAD

**Issue by fabriziobertocci** [GaloisInc/daedalus#103]
In the latest version of Deadlus the variable `Offset` appears to contain the number of remaining bytes in the stream (offset from the end) and not the offset from the beginning of the stream.

This breaks the definition of the following parser that is used to align the stream to a certain number of bytes:
```
def PadTo n = {
    @here = Offset;
    { (here % n == 0) is true; ^ {} } <| @(Many (n - (here % n))  UInt8);
}
```
Is this an intended change? If so how do we get access to the offset since the beginning of the stream?

**Comment by yav** [comment]
No this was a bug, sorry about that.   I fixed the issue on the current `master` (a3498dd919ca325a22), let me know if that soled the problem, and we can close the ticket.

Just FYI, when in doubt, you can try the `--core` flag to `daedalus`.  This uses a different interpreter that should behave exactly like the normal one.   Please let us know if you ever notice a different behavior between those two, as this is a fairly strong indication of a bug.

**Comment by fabriziobertocci** [comment]
Ok I have verified the latest version (from master) have the fix. Thanks
